spec.html

```diff
@@ -9376,10 +9376,7 @@ <h1>[[HasProperty]] ( _P_ )</h1>
             1. If _numericIndex_ is not *undefined*, then
               1. Let _buffer_ be _O_.[[ViewedArrayBuffer]].
               1. If IsDetachedBuffer(_buffer_) is *true*, throw a *TypeError* exception.
-              1. If IsInteger(_numericIndex_) is *false*, return *false*.
-              1. If _numericIndex_ = *-0*, return *false*.
-              1. If _numericIndex_ &lt; 0, return *false*.
-              1. If _numericIndex_ &ge; _O_.[[ArrayLength]], return *false*.
+              1. If IsIntegerIndexValid(_O_, _numericIndex_) *false*, return *false*.
```

Comment:
```suggestion
              1. If IsIntegerIndexValid(_O_, _numericIndex_) is *false*, return *false*.
```

Reply:
I can't commit the suggestion for some reason, will force-push.